community/kernel/src/main/java/org/neo4j/helpers/Format.java

```diff
@@ -35,10 +35,12 @@
      * sources is an easier task.
      */
     public static final TimeZone DEFAULT_TIME_ZONE = TimeZone.getTimeZone( "UTC" );
-    public static final String DATE_FORMAT = "yyyy-MM-dd HH:mm:ss.SSSZ";
-    public static final String TIME_FORMAT = "HH:mm:ss.SSS";
 
     private static final String[] BYTE_SIZES = { "B", "kB", "MB", "GB" };
+    private static final String[] COUNT_SIZES = { "", "k", "M", "bn", "tr" };
```

Comment:
At least case should be aligned between different count sizes

Reply:
https://en.wikipedia.org/wiki/Metric_prefix#List_of_SI_prefixes as you can see they aren't all the same case. Perhaps we should just conform to that instead, which means it should be:

`"", "k", "M", "G", "T"`

Reply:
ok sounds good